core/geometry/src/geometry3d/Transform.ts

```diff
@@ -21,20 +21,18 @@ import { TransformProps, XAndY, XYAndZ } from "./XYZProps";
  * * The math for a Transform `T` consisting of a Matrix3d `M` and a Point3d `o` on a Vector3d `p` is: `Tp = M*p + o`.
  * In other words, `T` is a combination of two operations on `p`: the action of matrix multiplication, followed by a
  * translation. `Origin` is a traditional term for `o`, because `T` can be interpreted as a change of basis from the
- * global axes centered at the global origin, to a new set of axes centered at `o`.
- * * Beware that for common transformations (e.g. scale about point, rotate around an axis, mirror across a
- * plane) the "fixed point" that is used when describing the transform is NOT the "origin" stored in the
- * transform. Setup methods (e.g createFixedPointAndMatrix, createScaleAboutPoint) take care of determining
- * the appropriate origin coordinates.
+ * global axes centered at the global origin, to a new set of axes specified by matrix M columns centered at `o`.
+ * * Beware that for common transformations (e.g. scale about point, rotate around an axis) the `fixed point` that
+ * is used when describing the transform is NOT the `origin` stored in the transform. Setup methods (e.g
+ * createFixedPointAndMatrix, createScaleAboutPoint) take care of determining the appropriate origin coordinates.
+ * * If `T` is a translation, no point is fixed by `T`.
+ * * If `T` is the identity, all points are fixed by `T`.
+ * * If `T` is a scale about a point, one point is fixed by `T`.
+ * * If `T` is a rotation about an axis, a line is fixed by `T`.
+ * * If `T` is a projection to the plane, a plane is fixed by `T`.
```

Comment:
nice!

Reply:
Thanks! That's one of your chat comments!